tools/cockpit.spec

```diff
@@ -175,67 +200,121 @@ install -p -m 644 cockpit.pp %{buildroot}%{_datadir}/selinux/targeted/
 %ifnarch x86_64
 rm -rf %{buildroot}/%{_datadir}/%{name}/docker
 %endif
+%if 0%{?suse_version}
+%suse_update_desktop_file -u -r -G 'Cockpit Server Manager' %{name} System Utility
+mv %{buildroot}%{_datadir}/doc/%{name} %{buildroot}%{_docdir}/%{name}/html
+%endif
 
 %files
+%if 0%{?suse_version}
+%defattr(-,root,root)
```

Comment:
I'd wish this would be the default for SUSE. :-/
Yes, all SUSE rpms need this in each %files section :-(

I need to talk to the release manager (resp. the rpm maintainer) why this isn't the default.